client/src/AuthenticatedApp/Calculators/CardLink.tsx

```diff
@@ -0,0 +1,32 @@
+import { Button, CardContent } from '@mui/material';
+import { Link, LinkProps } from 'react-router-dom';
+import { forwardRef } from 'react';
+import LayersOutlinedIcon from '@mui/icons-material/LayersOutlined';
+import { CardContainer } from './CardContainer';
+
+const CustomLink = forwardRef<HTMLAnchorElement, LinkProps>(
```

Comment:
there is CustomLink in NavLink , try reuse it

Reply:
+1